### PR TITLE
thread safe bug fixed.

### PR DESCRIPTION
**Pull request by recepdmr**

### CODE REVIEW

**Comment by peacecwz** [comment]
@recepdmr You have conlicts, can you fix conflicts?

**Review by peacecwz — CHANGES_REQUESTED** [review]
@recepdmr You have conlicts, can you fix conflicts?

**Comment by recepdmr** [comment]
@peacecwz ok. I will fix.

File: src/Koledeus.Client/Services/ICPUInfoService.cs [hunk 1]
```diff
@@ -16,16 +16,16 @@ public class ProcessInfoService : ICPUInfoService
             double totalCpuUsage = 0;
             long totalMemoryUsage = 0;
             var processes = Process.GetProcesses();
-
-            // TODO (peacecwz): This code is not working on threadsafe, It needs refactoring for working threadsafe
-            Parallel.ForEach(processes, process =>
+            
+            foreach (var process in processes)
```
Comment (peacecwz) [inline src/Koledeus.Client/Services/ICPUInfoService.cs:20]: It doesn't solve it in a better way. You need to use Parallel working and handling race condition with Parallel.Foreach :)

**Review by peacecwz — CHANGES_REQUESTED** [review]

**Pull request closed without merging**